bundles/org.openhab.io.openhabcloud/src/main/java/org/openhab/io/openhabcloud/internal/CloudClient.java

```diff
@@ -161,9 +173,9 @@ public CloudClient(HttpClient httpClient, String uuid, String secret, String bas
         this.remoteAccessEnabled = remoteAccessEnabled;
         this.exposedItems = exposedItems;
         this.jettyClient = httpClient;
-        reconnectBackoff.setMin(1000);
-        reconnectBackoff.setMax(30_000);
-        reconnectBackoff.setJitter(0.5);
+        reconnectBackoff.setMin(RECONNECT_MIN);
```

Comment:
Perhaps it would make sense to have larger initial delay, to be less aggressive to server

Reply:
[I suggested](https://github.com/openhab/openhab-addons/issues/12121#issuecomment-1377539222) longer delays:  `RECONNECT_MIN=2000`, `jitter=0.75`. Capping to 60 seconds makes sense.

![image](https://user-images.githubusercontent.com/315866/215263082-c829dd1f-b80b-41ec-b9c2-c2c4d5d1e557.png)

```
attempt min/max
0 [500.0, 3500.0],
1 [1000.0, 7000.0],
2 [2000.0, 14000.0],
3 [4000.0, 28000.0],
4 [8000.0, 56000.0],
5 [16000.0, 60000]
```

Compare this to this PR (`RECONNECT_MIN=1000`, `JITTER=0.5`)

```
attempt min/max
0 [500.0, 1500.0],
1 [1000.0, 3000.0],
2 [2000.0, 6000.0],
3 [4000.0, 12000.0],
4 [8000.0, 24000.0],
5 [16000.0, 48000.0]]
```
![image](https://user-images.githubusercontent.com/315866/215263170-e59679d3-2fa4-41d1-96a8-240b0c6b6e36.png)

Reply:
Agreed

Reply:
Yeah , that seems more reasonable.  Let give it a go.